roles/overpass-api/templates/overpass-areas-dispatcher.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Overpass areas dispatcher
+AssertPathExists={{ overpass_database_dir }}/replicate_id
+
+[Service]
+Type=simple
+User={{ overpass_user }}
+ExecStartPre=-/bin/rm {{ overpass_database_dir }}/osm3s_{{ overpass_version }}_areas
```

Comment:
Can we remove those rm ... ExecStartPre? See http://wiki.openstreetmap.org/w/index.php?title=Overpass_API/Installation&diff=1502925&oldid=1459308&rcid=&curid=72215: 

> Do not automatically remove any of the lock files, socket files or shared memory files. They work as canaries, i.e. hitting existing files is almost always an indicator for bigger trouble elsewhere. Please ask back in those cases.

(this affects all dispatcher scripts)

Reply:
Make sense. These PreExecStart statements are from wiki but I didn't check if they're usefull or needed. Thanks for clarification.

Reply:
Those wiki scripts were added by third party users, have never been reviewed and caused serious issues down the road (segmentation fault). Roland puts extra scrutiny on those now, making sure they're ok.